src/test/java/strategy/BalanceOperationHendlerTest.java

```diff
@@ -0,0 +1,35 @@
+package strategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Map;
+import model.Fruit;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import storage.Storage;
+
+public class BalanceOperationHendlerTest {
+    private static OperationHandler balanceOperationHandler;
+    private static Map<Fruit, Integer> storage;
+
+    @BeforeClass
+    public static void beforeClass() {
+        balanceOperationHandler = new BalanceOperationHandler();
+        storage = Storage.data;

```

Comment:
remove this storage field and use Storage.data everywhere  (do the same in another test classes)